### ISSUE TITLE
FTX – how to distinguish a failed market order from a successful one?

### ISSUE THREAD

**Issue by theantiqa** [ccxt/ccxt#7283]
![image](https://user-images.githubusercontent.com/46623939/87634774-8e34d580-c75b-11ea-9b18-97a19e59d03b.png)

As you can see in above screen shot you can see i have place 2 order. and you can see there is 2 response one id failed order and second is success order. and you can see there are not any difference in response even how can i know order placed success or not, there are not any flag using that i can know. so can you tell me how can i get order status.

even how can i know at which price my order placed because it s market order. 

in short there have some information missing.

I hope, I will get response very soon.

Thanks

**Comment by kroitor** [comment]
@theantiqa can you please post your code you use to create the order? You can fetch the status of the order with `fetch_order` after `create_order`...

Also, please, don't post screenshots (images) of text, instead, always paste your source code and output here using [GitHub markdown](https://guides.github.com/features/mastering-markdown/) for readability and padding, surround your code and/or output with triple backticks (&#096;&#096;&#096;), like shown below, for example:

&#096;&#096;&#096;Python
put your code here – something we can copypaste and launch
don't paste one line of code, paste your full short complete code snippet to reproduce it
it has to be short but complete including exchange instantiation, but without your keys of course
&#096;&#096;&#096;

&#096;&#096;&#096;
your output here
&#096;&#096;&#096;

**Comment by kroitor** [comment]
@theantiqa in general, when you place an order with an exchange, the execution is not instantaneous. It takes time for the exchange to accept it and process it. This is documented in the Manual. 

In Python it would look like so:

```Python
import ccxt
from pprint import pprint

exchange = ccxt.ftx({
    'enableRateLimit': True,
    'apiKey': 'YOUR_API_KEY',
    'secret': 'YOUR_API_SECRET',
})

exchange.load_markets()
symbol = 'YOUR_SYMBOL'
type = 'market'
side = 'buy'
amount = YOUR_AMOUNT
price = None
params = {}

created_order = exchange.create_order(symbol, type, side, amount, price, params)
updated_order = exchange.fetch_order(created_order['id'])

pprint(updated_order)
print('Order status is:', updated_order['status'])
```

Let us know if the above does not answer your question. Feel free to reopen this issue or just ask further questions if any.

**Comment by theantiqa** [comment]
Thanks for your reply and i understand it.

can you give one more solution?
I want to place leverage order in ftx. how can i place order?

**Comment by kroitor** [comment]
@theantiqa i'll post an example here within a few hours, please, stand by.

**Comment by theantiqa** [comment]
thanks

**Comment by theantiqa** [comment]
Can you please give me Solution ASAP. because i am working on deadline project. its urgent.

**Comment by kroitor** [comment]
@theantiqa I will post an example, however, I can't promise you an ETA. This is an open-source project, and we have thousands of users all working on their deadline projects.

**Comment by theantiqa** [comment]
Can you tell me how to place stop loss order in ftx?

**Comment by kroitor** [comment]
@theantiqa 

> Can you tell me how to place limit stop loss order in ftx?

You do that just as you would do with a regular `create_order`, as explained in the Manual:

```Python
import ccxt
from pprint import pprint

exchange = ccxt.ftx({
    'enableRateLimit': True,
    'apiKey': 'YOUR_API_KEY',
    'secret': 'YOUR_SECRET',
})

symbol = 'BNB/USD'
type = 'stop'
side = 'sell'
amount = 0.4
price = 11
# your params overrides (if any) as explained in the Manual
params = {
}

exchange.load_markets()
order = exchange.create_order(symbol, type, side, amount, price, params)
pprint(order)
```

**Comment by theantiqa** [comment]
suppose want to trade in this symbol BTC/USD

current market price is = $9000 of 1 btc

i want to place limit order with price $9100 and with stop loss $8050.
then how to place order in ftx exchange?

**Comment by kroitor** [comment]
@theantiqa this is documented in the Manual: 

- https://github.com/ccxt/ccxt/wiki/Manual#overriding-unified-api-params

```Python
import ccxt
from pprint import pprint

exchange = ccxt.ftx({
    'enableRateLimit': True,
    'apiKey': 'YOUR_API_KEY',
    'secret': 'YOUR_SECRET',
})

exchange.load_markets()

# exchange.verbose = True  # uncomment for debugging if you have issues with it

symbol = 'BTC/USD'
type = 'stop'
side = 'sell'
amount = 1
stop_price = 8050
price = 9000
# your params overrides (if any) as explained in the Manual
params = {
    'orderPrice': exchange.priceToPrecision(symbol, price)
}

order = exchange.create_order(symbol, type, side, amount, price, params)
pprint(order)
```

**Comment by theantiqa** [comment]
Thanks 
Its done. 
You are great.
